ddtrace/tracer/span_test.go

```diff
@@ -619,6 +619,20 @@ func TestSpanLog(t *testing.T) {
 	})
 }
 
+func TestNormalizeSpan(t *testing.T) {
+	t.Run("negativeduration", func(t *testing.T) {
+		actual := span{
+			Service:  "testservice",
+			Duration: -1,
+		}
+		normalizeSpan(&actual)
```

Comment:
normalizeSpan is too trivial for this to be valuable. Don't we have tests for `aggregableSpan`?